__tests__/app.test.js

```diff
@@ -0,0 +1,70 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("/general errors", () => {
+  test("GET:404 responds with 404: not found for a non-existent path", () => {
+    return request(app)
+      .get("/not-a-path")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("path not found");
+      });
+  });
+  test("POST:406 responds with 406 when POST request receieved for path where method is not allowed", () => {
+    return request(app)
+      .post("/api/topics")
+      .expect(406)
```

Comment:
All of these 406 tests can be removed as we will be introducing these methods later on which means that these tests will fail.